agent/pkg/api/socket_data_streamer.go

```diff
@@ -79,7 +79,9 @@ func (e *BasenineEntryStreamer) Get(ctx context.Context, socketId int, params *W
 	go handleDataChannel(connection, data)
 	go handleMetaChannel(connection, meta)
 
-	connection.Query(query, data, meta)
+	if err = connection.Query(query, data, meta); err != nil {
+		logger.Log.Errorf("Query mode call failed: %v", err)
```

Comment:
We don't need to return nil/switch to panic?

Reply:
I'm afraid of crashing the API server based on user action but I agree that `Panicf` is required there. So, fixed with https://github.com/up9inc/mizu/commit/296b924460171166021e2c7b4c4a163594829713